admin/acceptance.py

```diff
@@ -106,6 +106,14 @@ def start_nodes(self):
             cwd=self.vagrant_path.path,
             env=extend_environ(FLOCKER_BOX_VERSION=box_version))
 
+        # Import this here, since `run` isn't a good generic name.
+        from flocker.provision._install import run, task_pull_docker_images
```

Comment:
How about putting imports at top and doing `import run as run_on_provisioned_node`?